.github/workflows/progress-tracker.yml

```diff
@@ -77,55 +120,75 @@ jobs:
+
             const taskIssues = await github.paginate(github.issues.listForRepo, {
               owner: context.repo.owner,
               repo: context.repo.repo,
               direction: 'asc',
               labels: filterLabel
             })
 
-            let report = ''
-            let chapters = 0
+            let chapters = []
             for (const issue of taskIssues) {
               const tasksStatus = parseTasks(issue.body)
               const tasksCount = tasksStatus.length
               if(tasksCount != totalTrackedTasks) {
                 core.info(`Skipping issue #${issue.number} with ${tasksCount} instead of ${totalTrackedTasks} tasks`)
                 continue
               }
-              if(!report) {
-                report = `Chapters | Links | Team | ${tasksStatus.map((v, i) => i + 1).join(' | ')}\n`
-                report += `:--------|:------|:----:|:${tasksStatus.map(() => '-').join(':|:')}:\n`
-              }
-              const links = formatLinks(parseLinks(issue.body))
-              const team = formatTeam(parseTeam(issue.body))
               core.info(`Adding issue #${issue.number} with ${tasksCount} tasks`)
-              chapters++
-              report += `[${issue.title.replace(/\s+\d+$/, '')}](${issue.html_url}) | ${links} | ${team} | ${tasksStatus.join(' | ')}\n`
+              const heading = parseHeading(issue.body)
+              chapters.push({
+                id: issue.number,
+                url: issue.html_url,
+                title: heading.title || issue.title,
+                part: heading.part,
+                number: heading.number,
+                links: parseLinks(issue.body),
+                team: parseTeam(issue.body),
+                tasks: tasksStatus
+              })
             }
 
-            const progressIssue = await github.issues.get({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              issue_number: trackerIssueNumber
+            if(!chapters.length) {
+              core.info(`Nothing to report`)
+              return
+            }
+
+            chapters.sort((a, b) => a.number.localeCompare(b.number, undefined, {numeric: true}))
+
+            chapters.forEach(c => {
+              Object.entries(c.team).forEach(([k, v]) => {
+                v.forEach(m => {
+                  teamMembers[k][m] = (teamMembers[k][m] || []).concat(c.title)
+                })
+              })
             })
 
+            const report = [
+              `Chapters | <span title="${Object.entries(icons).map(([k, v]) => `${v} (${k})`).join(', ')}">Links</span> | <span title="${Object.keys(teamMembers).join('/')}">Team</span> | ${trackedTasks.map((v, i) => `<span title="${v}">${i + 1}<span>`).join(' | ')}`,
+              `:--------|:------|:----:${'|:-:'.repeat(totalTrackedTasks)}`
+            ].concat(chapters.map(formatRow)).join('\n')
+
+            const teams = Object.entries(teamMembers).map(([k, v]) => {
```

Comment:
With ~100 contributors, I think this section would get too unwieldy, and it's also not necessary to "@" notify each contributor.

I still like the idea of showing some aggregate stats. Could we do something like this instead?

### Contributor stats

  | Authors | Reviewers | Analysts
-- | -- | -- | --
Total | 30 | 45 | 20
Min | 1 | 1 | 0
Max | 3 | 9 | 2
Median | 2 | 3 | 1
Average | 2.5 | 5 | 1.1

Reply:
> With ~100 contributors, I think this section would get too unwieldy

I thought about it, but I decided to add it anyway because there is one piece of information that is not readily available anywhere, "in how many chapters a specific member is involved in and in what ways?", which I think is an important thing to know to properly distribute workload.

> and it's also not necessary to "@" notify each contributor.

That's a very good point, I will certainly remove auto-mention feature.

> I still like the idea of showing some aggregate stats. Could we do something like this instead?

Summarized statistical table is a great idea, but we can perhaps do it in a separate iteration (due to time constraints at my end).

Reply:
@rviscomi I think I have taken care of the issue of getting too unwieldy by hiding the list under a <details> element, so it will be there, but won't bother.

Reply:
Rather than making the contributors a `<details>` element could you not make each sub-team the `<details>` element? So can always see the team counts but can expand for names in a team.

Reply:
> Rather than making the contributors a `<details>` element could you not make each sub-team the `<details>` element? So can always see the team counts but can expand for names in a team.

We can do it, but I think the current approach is more friendly in some ways (I am open to be convinced otherwise):

* It is a compromise between clicking vs. scrolling, and right now individual contributor usernames are not in a separate line as bullet points, so I do not thing there will be too much scrolling (perhaps we will have a better idea of how much space it takes once it is merged here and reports are generated).
* Currently, contributors report is composed as "h3" headings for each team name followed by a paragraph of comma separated usernames, which is independent of how it is finally placed in the report, i.e., only at the very end we decide to place it in a "details" container (which can be something else too), but if we decide it otherwise, "details" and "summary" elements will be engraved during the iteration over the data.

It may also depend on how the information is used often, a specific role at a time or a specific person (across rolls) at a time.

Reply:
@rviscomi I have added stats table as well. Please have a look at https://github.com/bazzadp/almanac.httparchive.org/issues/73 for an example.